pkg/client/version_check.go

```diff
@@ -29,61 +29,45 @@ type ApiVersionObject struct {
 	Version string `json:"version"`
 }
 
-const (
-	versionPathClassic  = "/api/v1/config/clusterversion"
-	versionPathPlatform = "/platform/core/v1/version"
-)
-
-// EnvironmentType represents the type / generation of an environment
-type EnvironmentType int
-
-const (
-	// Classic identifies a Dynatrace Classic environment
-	Classic EnvironmentType = iota
-
-	// Platform identifies a Dynatrace Platform environment
-	Platform
-)
-
-// Environment represents a Dynatrace environment
-type Environment struct {
-	// URL is the base URL of the environment
-	URL string
-	// Type is the type / generation of environment
-	Type EnvironmentType
-}
+const versionPathClassic = "/api/v1/config/clusterversion"
 
 // GetDynatraceVersion returns the version of an environment
-func GetDynatraceVersion(client *http.Client, environment Environment) (version.Version, error) {
-	var versionURL string
-	switch environment.Type {
-	case Classic:
-		versionURL = environment.URL + versionPathClassic
-	case Platform:
-		versionURL = environment.URL + versionPathPlatform
-	default:
-		return version.Version{}, fmt.Errorf("usupported environment type")
-	}
+func GetDynatraceVersion(client *http.Client, environmentURL string) (version.Version, error) {
+	versionURL := environmentURL + versionPathClassic
+
 	resp, err := rest.Get(client, versionURL)
 	if err != nil {
 		return version.Version{}, fmt.Errorf("failed to query version of Dynatrace environment: %w", err)
 	}
 	if resp.StatusCode != http.StatusOK {
```

Comment:
```suggestion
	if !resp.IsSuccess() {
```

just saw it here as well